src/components/app/userActionFormVotingInformationResearched/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const ANALYTICS_NAME_USER_ACTION_FORM_VOTING_INFORMATION_RESEARCHED =
+  'User Action Form Voting Information Researched'
+
+export enum SECTIONS_NAMES {
+  LOADING = 'Loading',
+  ADDRESS = 'Address',
+  SUCCESS = 'Success',
+}
```

Comment:
Let's keep the pattern for enum naming, it should be pascal case and all the values in constant case.
```suggestion
export enum SectionsNames {
  LOADING = 'Loading',
  ADDRESS = 'Address',
  SUCCESS = 'Success',
}
```

Reply:
Gonna add it to eslint